src/components/Editor/Tabs.css

```diff
@@ -1,58 +1,53 @@
 .source-header {
   border-bottom: 1px solid var(--theme-splitter-color);
+  width: 100%;
   height: 30px;
   display: flex;
-  flex: 1;
-  flex-flow: row wrap;
-  -webkit-align-items: stretch;
-  align-items: stretch;
+  justify-content: space-between;
 }
 
 .source-header * {
   -moz-user-select: none;
   user-select: none;
 }
 
-.source-tabs {
-  max-width: calc(100% - 80px);
-  float: left;
-  margin-inline-start: 21px;
-}
-
 .source-header .new-tab-btn {
-  width: 14px;
-  height: 14px;
-  display: inline-block;
-  position: relative;
-  top: 4px;
-  margin: 4px;
-  margin-inline-start: 8px;
-  line-height: 0;
+  align-self: flex-end;
+  padding: 4px 2px;
   cursor: pointer;
   fill: var(--theme-comment);
   transition: 0.1s ease;
 }
 
+.source-header .new-tab-btn svg {
+  width: 12px;
+}
+
+.source-tabs {
+  max-width: calc(100% - 80px);
+  align-self: flex-end;
```

Comment:
I think we shouldn't have this here. In fact, removing it fixes the issue with the "+" and ">>" buttons disapearing